src/features/landing/components/AuthSection/AuthForm/constants.ts

```diff
@@ -0,0 +1,73 @@
+import { faker } from '@faker-js/faker';
+
+import { Mock } from './types';
+
+export const fullNamePlaceholder: RegExp = /Mykhailo Svitskyi/;
+export const emailPlaceholder: string = 'vilnaCRM@gmail.com';
+export const passwordPlaceholder: string = 'Create a password';
+
+export const submitButtonText: string = 'Sign-Up';
```

Comment:
we have constants and mocks in one file, this is two different data that must be separated

Reply:
Fixed